app/views/curation_concerns/base/_form_progress.html.erb

```diff
@@ -1,44 +1,46 @@
-    <aside class="form-progress panel panel-default" id="form-progress">
-      <div class="panel-heading">
-        <h2><%= t("sufia.#{controller_name}.progress.header") %></h2>
+<div id="form-progress">
```

Comment:
Ah. A remnant of another technique where I was trying to call a parent div and ended up using an ID from the template. Will remove. Thx